src/tokens/hooks.ts

```diff
@@ -121,11 +121,9 @@ export function useCashInTokens() {
   return useSelector((state) => cashInTokensByNetworkIdSelector(state, networkIdsForCico))
 }
 
-export function useCashOutTokens(showZeroBalanceTokens: boolean = false) {
```

Comment:
cKES isn't marked as `showZeroBalanceTokens` and don't want to change it in address-metadata because it will mess with Valora. Since only cKES and cUSD exist in the mento world we don't need these extra filters because both should be shown in the 0 state.